vars/Debian.yml

```diff
@@ -1,4 +1,5 @@
 ---
+nginx_package: nginx
```

Comment:
Yes it is, contains additional configurations[Nginx](https://packages.debian.org/jessie/nginx) and [nginx-extras](https://packages.debian.org/jessie/nginx-extras)